assets/components/paymentMethods/paymentMethods.jsx

```diff
@@ -18,14 +19,15 @@ type PropTypes = {
   payPalButtonExists: boolean,
   stripeCallback: Function,
   payPalCallback: Function,
+  processing: boolean,
```

Comment:
No flow error because the types get lost on the way, but I've added a default value in oneoff-contributions/.../paymentMethodsContainer.jsx to be more explicit